corehq/apps/domain/forms.py

```diff
@@ -1168,6 +1168,7 @@ class DomainInternalForm(forms.Form, SubAreaMixin):
     )
 
     def __init__(self, domain, can_edit_eula, *args, **kwargs):
+        self.user = kwargs.pop('user', None)
```

Comment:
nit: Newer versions of Python have syntax for keyword-only arguments.
```suggestion
    def __init__(self, domain, can_edit_eula, *args, user=None, **kwargs):
        self.user = user
```